.circleci/config.yml

```diff
@@ -5,43 +5,120 @@ version: 2.1
 # Executors
 # -------------------------------------------------------------------------------------
 executors:
-  gpu:
+  gpu_cuda10:
```

Comment:
Maybe we should rebase on mau.update_ci so you don't have to repeat changes to this file?

Reply:
this branch is already based on top of `mau.update_ci`, but here it's showing the diff to `main` and that's unfortunate... it doesn't make sense to have this being an independent branch as the changes here require the ci, do you think there is a better way? should I target `mau.update_ci` as merge target? this doesn't look ideal either.

Reply:
Hmm, if there isn't an easy way to set this up with git then it makes sense to leave as is. Maybe once the other PR is merged this gets resolved.

Reply:
makes sense to me - so since now on the target with a set of subsequent diffs (as I have here) is setting as target the parent branch?

Reply:
Done, now it show as only change in the CI the new GPU tests added to the list of GPU tests.

Reply:
I still see diff for changes that were on the other brach and already merged. Did GitHub get confused here?

Reply:
probably! It wasn't showing those changes when I changed the target.
Probably it understood that the new target for merge is main (after update_ci was merged), but it's showing the diff to the old main! Bug report to GitHub?

Reply:
merged the new "main" into this branch, now it's showing the correct diff.

Reply:
Not sure what the correct behavior of github should be, should it show the diff merged-branch-with-main to current main, of current branch to common ancestor, where the branch branched? probabaly github is doing the right thing after all.

Reply:
I think it should show the diff with whatever the current state of main is, otherwise things can get really confusing. In the past when this happens, I change the target to some other branch, then back to main, and it solves it.

Reply:
so it cannot do a plain diff do main, or main could have included a ton of unrelated stuff from other sides. the automatic merge didn't work (there was a small conflict, no idea of why, considering that main was not identical to the previous target branch!), so it's justfied that it showed branch-to-branch-point instead of merged-branch-to-main diff.

Reply:
> I change the target to some other branch, then back to main, and it solves it
This has worked for me too.